packages/@aws-cdk/aws-ivs/test/integ.ivs.ts

```diff
@@ -0,0 +1,34 @@
+import { App, CfnOutput, Stack } from '@aws-cdk/core';
+import * as ivs from '../lib';
+
+const app = new App();
+
+const stack = new Stack(app, 'aws-cdk-ivs');
```

Comment:
can you include stack verification steps as described in the [CONTRIBUTING](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#step-3-work-your-magic) guide?

Reply:
Are the steps just CLI commands that don't error? How are these checked? One of the examples in the guide is broken.

Reply:
they should include the steps to verify that not just the deploy,but the resource has been configured as expected.
here's an [example](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-stepfunctions-tasks/test/lambda/integ.invoke.ts#L6-L17) you can follow.

i'll fix up the broken link.